modules/echo.ts

```diff
@@ -7,7 +7,6 @@ export const commands = [
     secret: false, // optional, default false unless permissionLevel is 3, then it's true
     description: 'Takes your input and spits it right back at you.', // required if not secret
     examples: ['yes'], // required if not secret
-    permissionLevel: 0, // optional, default 0 | 0: anyone, 1: server mods, 2: server admins, 3: bot admins
```

Comment:
The `echo` module is our (for the moment) base documentation for future modules. What would be the correct way of detailing module permissions then?

Reply:
Added a example and a link to a list of all available permissions.